### DISCUSSION TITLE
Updated Nginx, persistent data lost?

### DISCUSSION THREAD

**Discussion by kosymodo** [NginxProxyManager/nginx-proxy-manager#2847] [Q&A]
I've attempted to update Nginx today, as I've done on occasions before.  However, seem to have hit a problem today :(

It appears that I've lost all my persistent data (i.e. proxy hosts, SSL certs) in addition to my user details.  Not sure why this has happened, as all I did was to stop my nginx_app_1 container, recreated it (pulling latest image), then started it again.  Weird thing though is that all my proxy hosts themselves seem to still be working (e.g. nginx.MYDOMAIN.co.uk still resolves to 192.168.1.200:81) and my custom port configs appear intact.

Any ideas what I could try to do to recover the situation?

**Comment by wash3re** [comment]
I am having the same problem.  I even tried to do the SQLite password reset, and created a new account.  I was able to get in, but nothing shows up (poxies, certs, etc.).  But it is all still working.  I need to update some certs, so I kinda need to get it working somehow.

**Comment by MisterrrX** [comment]
Has there been any updates on this topic? Facing the same issue after updating.
